projects/storefrontlib/src/lib/checkout/components/multi-step-checkout/payment-method/payment-method.component.scss

```diff
@@ -1,128 +1,66 @@
-  }
-  &__billing {
-    margin-bottom: $cx-payment-method-billing-margin-bottom;
-    &-address {
-      @include type('4');
-      padding: $cx-payment-method-billing-address-padding;
-    }
-  }
-  &__img {
-    margin: $cx-payment-method-img-margin;
-    &-container {
-      display: flex;
-    }
+
+  @include checkout-media-style();
+}
+
+.cx-payment-title {
+  text-transform: var(--cx-text-transform, uppercase);
+  margin: var(--cx-margin, 0 auto);
+  padding: var(--cx-padding, 2.375rem 0 1.75rem 0);
+}
+
+.cx-payment-text {
+  @include media-breakpoint-down(md) {
+    padding-left: var(--cx-padding, 3.5rem);
   }
-  &__title {
-    text-transform: uppercase;
-    margin: $cx-payment-method-title-margin;
-    padding: $cx-payment-method-title-padding;
+  @include media-breakpoint-down(sm) {
+    padding-left: var(--cx-padding, 1.5rem);
   }
-  &__spinner {
-    padding-top: $cx-payment-methods-loading-padding;
-    padding-bottom: $cx-payment-methods-loading-padding;
+}
+
+.cx-payment-btn {
+  padding-bottom: var(--cx-padding, 1rem);
+
+  @include media-breakpoint-down(md) {
+    padding: var(--cx-padding, 0 3.5rem 1.25rem 0);
   }
-  &__body {
-    display: flex;
-    align-items: flex-end;
-    padding-bottom: $cx-payment-method-body-padding-bottom;
+  @include media-breakpoint-down(sm) {
+    padding: var(--cx-padding, 1.25rem);
   }
-  // ***************************** MEDIA QUERIES *****************************
-  @include checkout-media-style();
-  &__body {
-    &.row {
-      @include media-breakpoint-down(md) {
-        padding: $cx-payment-method-no-padding;
-      }
-    }
-  }
-  &__text {
-    @include media-breakpoint-down(md) {
-      padding-left: $cx-payment-method-text-md-padding-left;
-    }
+
+  .btn-action {
     @include media-breakpoint-down(sm) {
-      padding-left: $cx-payment-method-text-sm-padding-left;
+      margin-bottom: var(--cx-margin, 1rem);
     }
   }
-  &__header {
-    @include media-breakpoint-down(md) {
-      padding: $cx-payment-method-no-padding;
-    }
-    &-image {
-      @include media-breakpoint-down(md) {
-        margin-right: $cx-payment-method-header-image-md-margin-right;
-        margin-left: $cx-payment-method-header-image-md-margin-left;
-      }
-    }
-  }
-  &__card {
+}
+
+.cx-payment-body {
+  &.row {
     @include media-breakpoint-down(md) {
-      @include var-color(
-        'background-color',
-        $cx-payment-method-card-background-color
-      );
+      padding: var(--cx-padding, 0);
     }
   }
-  &__btns {
-    .btn-action {
-      @include media-breakpoint-down(sm) {
-        margin-bottom: $cx-payment-method-btns-margin-bottom;
-      }
-    }
+}
+
+.cx-payment-card {
+  @include media-breakpoint-down(md) {
+    background-color: var(--cx-background-color, var(--cx-g-color-inverse));
   }
 }
+
+.cx-payment-spinner {
+  padding-top: var(--cx-padding, 30px 0);
+}
+
+.cx-payment-body {
+  display: var(--cx-display, flex);
```

Comment:
Could you reference the payment-body class only once? Or else you wont be able to merge.